plugins/BEdita/Core/src/ORM/Locator/TableLocator.php

```diff
@@ -37,10 +37,6 @@ class TableLocator extends CakeLocator
      */
     protected function _getClassName($alias, array $options = [])
     {
-        if (empty($options['className'])) {
-            $options['className'] = Inflector::camelize($alias);
-        }
-
```

Comment:
but is probably wiser to leave it, as it is in base class implementation... i'll do it